NuRadioReco/modules/io/coreas/coreas.py

```diff
@@ -122,137 +627,23 @@ def calculate_simulation_weights(positions, zenith, azimuth, site='summit', debu
 
     return weights
 
-
-def make_sim_station(station_id, corsika, observer, channel_ids, weight=None):
```

Comment:
this function is also deprecated? Shall we add a warning?

Reply:
Yes, I can do that.